index.js

```diff
@@ -2,8 +2,11 @@
 
 const main = require('./build/lib/main.js');
 
-var args = require('minimist')(process.argv.slice(2), {base: 'string'});
+var args =
+    require('minimist')(process.argv.slice(2), {base: 'string', boolean: ['semantic-diagnostics']});
 try {
+  args.semanticDiagnositcs = args['semantic-diagnostics'] || false;
+  delete args['semantic-diagnostics'];
```

Comment:
this seems like a clumsy way to handle args. How are the args passed over to the options list? I would handle the default value at that point instead.

Reply:
Oh here I was just trying to follow naming conventions so that the cli flag would be --semantic-diagnostics but the property in TranspilerOptions would be semanticDiagnostics. Once the args get passed into the Transpiler constructor they're expected to be of type TranspilerOptions, so it's no longer possible to access args['semantic-diagnostics']. What do you think would be the best way to resolve this?